test/integration-tests/Seaport.test.ts

```diff
@@ -26,6 +28,69 @@ describe('Seaport', () => {
   let permit2: Permit2
   let planner: RoutePlanner
   let cryptoCovens: ERC721
+  let weth: Contract
+
+  // @dev We have to separate the ERC20 tests because we use orders taken from a much
+  // newer block. If we were to replace all of the example orders with newer ones,
+  // many existing tests would break.
+  describe('Seaport ERC20', () => {
```

Comment:
I dont think this should be before the global beforeEach block on line 29 bc that's going to be executed still for this `describe` block - and it doesnt feel very clear like this. So imo we should put this describe block _below_ that beforeEach

Reply:
or actually a neater way to do it would be to put the rest of the file inside a `Seaport ETH` describe block, so that that `beforeEach` isnt executed for these tests unnecessarily

Reply:
Sounds good, I'll separate it into a `Seaport ETH` describe I think that's cleaner for sure. I'll also apply it to the V2 tests below